MediaBrowser.Common/Net/IPData.cs

```diff
@@ -0,0 +1,76 @@
+using System.Net;
+using System.Net.Sockets;
+using Microsoft.AspNetCore.HttpOverrides;
+
+namespace MediaBrowser.Common.Net
+{
+    /// <summary>
+    /// Base network object class.
+    /// </summary>
+    public class IPData
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="IPData"/> class.
+        /// </summary>
+        /// <param name="address">An <see cref="IPAddress"/>.</param>
+        /// <param name="subnet">The <see cref="IPNetwork"/>.</param>
+        public IPData(
+            IPAddress address,
+            IPNetwork? subnet)
+        {
+            Address = address;
+            Subnet = subnet ?? (address.AddressFamily == AddressFamily.InterNetwork ? new IPNetwork(address, 32) : new IPNetwork(address, 128));
+            Name = string.Empty;
```

Comment:
Normally you'd just use ` : this(address, subnet, string.Empty` or simply use `""` as the default value and skip the extra ctor entirely